Traceback (most recent call last):
  File "/home/panxopanza/miniconda3/lib/python3.11/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/panxopanza/miniconda3/lib/python3.11/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/panxopanza/miniconda3/lib/python3.11/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/panxopanza/miniconda3/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/home/panxopanza/miniconda3/lib/python3.11/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/home/panxopanza/miniconda3/lib/python3.11/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/panxopanza/miniconda3/lib/python3.11/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import numpy as np
from numpy import radians as rad # convertimos grados a radianes
import matplotlib.pyplot as plt
from empylib.waveoptics import multilayer
from empylib.ref_spectra import AM15
from empylib.ref_spectra import color_system as cs
cs = cs.hdtv

# Reflectividad en capa delgada
lam = np.linspace(0.3,0.8,100)   # longitud de onda (en um)
n_layers = (1.0,1.5,4.3)         # índices de refracción n1, n2, n3

cs.interp_internals(lam)
def color_R_film(d):
    Rp = lambda tt,d : multilayer(lam, rad(tt),n_layers, (d,), 'TM')[0]
    Rs = lambda tt,d : multilayer(lam, rad(tt),n_layers, (d,), 'TE')[0]
    
    # formateamos la figura
    fig, ax = plt.subplots()
    fig.set_size_inches(6, 4)
    plt.rcParams['font.size'] = '14'
    
    theta = np.linspace(0,90,100) # angulo de inciSdencia
    for tt in theta: 
        R = 0.5*Rp(tt,d) + 0.5*Rs(tt,d)
        Irad = R*AM15(lam)
        html_rgb = cs.spec_to_rgb(Irad, out_fmt='html')
        ax.axvline(tt, color=html_rgb, linewidth=6) 
    ax.set_xlim([min(theta),max(theta)])
    ax.set_ylim([0,1.0])
    ax.axes.yaxis.set_visible(False)
    ax.set_xlabel('Ángulo de incidencia (deg)')
    plt.show()
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mImportError[0m                               Traceback (most recent call last)
Cell [0;32mIn[9], line 6[0m
[1;32m      4[0m [38;5;28;01mfrom[39;00m [38;5;21;01mempylib[39;00m[38;5;21;01m.[39;00m[38;5;21;01mwaveoptics[39;00m [38;5;28;01mimport[39;00m multilayer
[1;32m      5[0m [38;5;28;01mfrom[39;00m [38;5;21;01mempylib[39;00m[38;5;21;01m.[39;00m[38;5;21;01mref_spectra[39;00m [38;5;28;01mimport[39;00m AM15
[0;32m----> 6[0m [38;5;28;01mfrom[39;00m [38;5;21;01mempylib[39;00m[38;5;21;01m.[39;00m[38;5;21;01mref_spectra[39;00m [38;5;28;01mimport[39;00m color_system [38;5;28;01mas[39;00m cs
[1;32m      7[0m cs [38;5;241m=[39m cs[38;5;241m.[39mhdtv
[1;32m      9[0m [38;5;66;03m# Reflectividad en capa delgada[39;00m

[0;31mImportError[0m: cannot import name 'color_system' from 'empylib.ref_spectra' (/home/panxopanza/miniconda3/lib/python3.11/site-packages/empylib/ref_spectra.py)

